ArjunaJTS/interop/glassfish/d.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+source init.sh
+set +e
+
+export PATH=$GLASSFISH/bin:$PATH
+[ -d "$JBOSS_HOME"  ] || fatal "file not found: $JBOSS_HOME"
+[ -f "$GLASSFISH/bin/asadmin"  ] || fatal "asadmin not found"
+
+WF_DEPLOY_DIR=$JBOSS_HOME/standalone/deployments
+
+usage() {
+  echo -e "$1: Usage: $0 [-a <gf|gf2|wf>] [-f <archive>] [-t <gfgf|gfwf|wfgf >] [-t <haltbefore|haltafter>]" 1>&2;
+  echo -e "examples:"
+  echo -e "\t$0 -t wfgf # send a message from wf to gf"
+  echo -e "\t$0 -t wfgf -t halt # send a message from wf to gf and then halt gf"
+  echo -e "\t$0 -a wf1 -f ${QS_DIR}/target/ejbtest.war # deploy an archive to wildfly"
+  echo -e "$0 -a gf1 -f recovery/target/dummy-resource-recovery.jar  # deploy an archive to WildFly"
+  exit 1;
+}
+
+http_get() {
+  res=$( curl --max-time 10 -qSfsw '\n%{http_code}' $1 ) # 2>/dev/null
+  exit_code=$?
+  http_code=$(echo "$res" | tail -n1) # the HTTP status code is the last line of the curl output
+
+  echo "http get for $1 curl status: $exit_code http status: $http_code res=$res"
+
+  if [ $exit_code = 0 ]; then
+    echo "http get for $1 curl ok"
+  else
+#    echo "SLEEP for 1 day (for debugging): curl error: $exit_code http status: $http_code for url $1"
```

Comment:
Agreed (I added this in order to debug a run inside a docker image and I have since fixed it)